source/index.html.md

```diff
@@ -408,6 +408,102 @@ This endpoint retrieves from a provided `organisation_id` all animals.
 | organisation_id  | ID of the organisation where the animal belongs to. | `required`       |
 | include_archived | Include archived animals.                           | default: `false` |
 
+## Put Animals
```

Comment:
to be consistent with the other subsections please change this to **Create/Update Multiple Animals**